estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,88 @@
+                <field name="bedrooms"/>
+                <field name="living_area"/>
+                <field name="expected_price"/>
+                <field name="selling_price"/>
+                <field name="date_availability"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="estate_property_view_form" model="ir.ui.view">
+        <field name="name">estate_property_form</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <form string="Estate Form View">
+                <sheet>
+                    <h1>
+                        <field name="name"/>
+                    </h1>
+                    <div class="d-flex justify-content-between">
+                        <div class="d-flex col">
+                            <group class="o_col">
+                                <field name="postcode"/>
+                                <field name="date_availability"/>
+                            </group>
+                        </div>
+                        <div class="d-flex col">
+                            <group class="o_col">
+                                <field name="expected_price"/>
+                                <field name="selling_price"/>
+                            </group>
+                        </div>
+                    </div>
+                    <notebook>
+                        <page string="Description">
+                            <div class="d-flex justify-content-space-between">
+                                <div class="col">
+                                    <group>
+                                        <field name="description"/>
+                                        <field name="bedrooms"/>
+                                        <field name="living_area"/>
+                                        <field name="facades"/>
+                                        <field name="garage"/>
+                                        <field name="garden"/>
+                                        <field name="garden_area"/>
+                                        <field name="garden_orientation"/>
+                                    </group>
+                                </div>
+                                <div class="col"/>
+                            </div>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="estate_property_view_search" model="ir.ui.view">
+        <field name="name">estate_property_search</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <search string="Estate">
+                <filter string="Available" name="available" domain="['|',('state','=','new'),('state','=','offer_received')]"/>
+                <filter string="Postcode" name="postcode" context="{'group_by':'postcode'}"/>
```

Comment:
```suggestion
                <filter string="Postcode" name="postcode" context="{'group_by' : 'postcode'}"/>
```

Reply:
I think it'd be more readable (and consistent with the commas) if we only place a space after the colon and not before it as well. Let me know if there's a rubric for this type of stuff.
```suggestion
                <filter string="Postcode" name="postcode" context="{'group_by': 'postcode'}"/>
```